frontend/public/components/container.tsx

```diff
@@ -351,28 +351,28 @@ const ContainerDetails: React.FC<ContainerDetailsProps> = (props) => {
 
       <hr />
 
-      <div className="row">
-        <div className="col-lg-4">
+      <>
```

Comment:
Are we able to remove this react fragment?

Reply:
sure can